updater/spec/dependabot/updater/operations_spec.rb

```diff
@@ -12,22 +12,6 @@
       Dependabot::Experiments.reset!
     end
 
-    it "returns nil if no operation matches" do
```

Comment:
Is there a reason this has been removed? What happens if no operation matches now?

Reply:
It still returns nil, but it's not possible for no operation to match at this point unless I feed it an invalid job.